quality_assurance.md

```diff
@@ -0,0 +1,93 @@
+# Quality Assurance for modules
+
+When a module creator is ready to request that their module be included, they will create a Pull Request (PR).  This begins the work of quality assurance.  As someone who is reviewing the modules created by others, it's important to have use a consistent method for evaluating content.
+
+## Step 1: Create an Issue
+
+* Click on "Issues" or go to https://github.com/arcus/education_modules/issues.
+* Choose "New Issue" (or if your screen is small, just "New").  This is a green button on the right side:
+![Screenshot of where to click for a new issue](media/new_issue.png)
+* Give your issue a good title: "QA" plus the proposed directory name from the PR. For example, if the PR includes a new module with the directory named "reproducibility", the title would be "QA reproducibility".)
+* In the "Write" tab area, paste the following (from `# Module Quality Assurance Report for PR #[put in the PR number here] ` to `* [ ] description or quote, line ___ in file ____`).  In square brackets, remove the square brackets and their contents and replace with the appropriate values.
```

Comment:
I also think we could mentioned the pinned sample issue here.

Reply:
done